src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -92,15 +86,17 @@ public void commandInserted(AWTEvent e) {
 
 	public static boolean checkUserGrants(String code) {
 
-		for (UserMenuItem umi : myMenu) {
+		Iterator<UserMenuItem> it = myMenu.iterator();
+		while (it.hasNext()) {
+			UserMenuItem umi = it.next();
 			if (umi.getCode().equalsIgnoreCase(code)) {
 				return true;
 			}
 		}
 		return false;
 	}
 
-	private int minButtonSize;
+	private int minButtonSize = 0;
```

Comment:
One doesn't have to init values to 0 or null as that is the default; doing the init again is redundant and unnecessary.

Reply:
It is not my code. Seems that I have only added and then removed this line:
//Object target = Context.getApplicationContext().getBean(Class.forName(app));
I will restore the previous version which is on infomatici.

Reply:
I restored the current informatici version